### ISSUE TITLE
Docs page enhancements

### ISSUE THREAD

**Issue by suntong** [cogentcore/core#1111]
> Interactive, editable examples of all major widgets are available on the main docs page (which is also a Cogent Core app running via wasm).

Here is what issues I found about our docs site.

- Copy & paste not possible, at least with my Linux Chrome. This makes it hard for people to follow the instructions as-is, and tiny typos will make getting started to have more hurdles.
- Search is not possible, when I type something in my search box and hit enter, I'm getting this:

![image](https://github.com/user-attachments/assets/ca766458-e7f2-42f8-bf19-22edd5df9339)

Meanwhile, where is the source text for our main docs page please?

**Comment by kkoreilly** [comment]
Thank you for filing this. The source for the docs is in the [docs](https://github.com/cogentcore/core/tree/main/docs) directory, mainly the [docs/content](https://github.com/cogentcore/core/tree/main/docs/content) directory. It is built using pages, which you can read about [here](https://www.cogentcore.org/core/widgets/other/pages). I am working on improving the search setup. I am also working on improving the selection support for copy and paste (see #1051).

**Comment by j0hnsmith** [comment]
I noticed that double clicking a word in the docs selects the whole paragraph instead of the word, also right click doesn't display the usual menu, seems like something is overriding default os behaviours.

**Comment by kkoreilly** [comment]
Yes, we are overriding default os behaviors, which is necessary for the event system to work correctly. We just need to improve text selection (#1051).

**Comment by suntong** [comment]
I just noticed that middle clicking which should open the underlying link in another tab is not working either.

**Comment by kkoreilly** [comment]
Yes, I will also work on fixing that.
